definitions.yaml

```diff
@@ -666,3 +666,84 @@ definitions:
       auth_token:
         type: string
         description: The newly created API token
+
+  # The list of all invitations.
+  V4InvitationsResponse:
+    type: array
+    description: List of members that belong to this organization
```

Comment:
this description sounds like it will be all members of the org, shouldn't this a list of all invites (no matter which status)?

Reply:
I know that is similar to all members, but all members sounds to me like excluding pending or expired invites